extensions/hazelcast-client/runtime/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-hazelcast-client-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-hazelcast-client</artifactId>
+    <name>Quarkus - Hazelcast - Client - Runtime</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast-all</artifactId>
+            <version>3.12.5</version>
```

Comment:
Add these versions to the bom/runtime/pom.xml BOM

Reply:
Also out of curiosity, do you need the `-all` dependency or can you use separate artifacts? It looks like it's 12Mb and I am not sure if all these classes are required, as they would impact the size of the resulting application

Reply:
In a new 4.0 release, we'll be merging some artifacts, but I believe for 3.0 we should be able to strip it down a bit

Reply:
That's no longer the case in 4.0, we are on the bare minimum.